testutils/clustermanager/perf-tests/main.go

```diff
@@ -44,7 +46,7 @@ func main() {
 		log.Fatal("Only one operation can be specified, either recreate or reconcile")
 	}
 
-	client, err := testPkg.NewClient()
+	client, err := testPkg.NewClient(gkeEnvironment)
```

Comment:
You're defaulting --gke-environment to an unsupported environment (empty string). NewClient() should either fail or the default value should be a valid one or the flag should be mandatory.

Reply:
Done.
Default the flag to be "prod".